backend/src/routers/analysis_router.py

```diff
@@ -21,10 +21,13 @@
 from ..models.phenotips_json import BasePhenotips
 from ..models.user import VerifyUser
 from ..security.security import get_authorization, get_current_user
+from . import analysis_discussion_router
 
 logger = logging.getLogger(__name__)
 
 router = APIRouter(prefix="/analysis", tags=["analysis"], dependencies=[Depends(database)])
+logger.info('including the api roter for discussions')
```

Comment:
```suggestion
```
Can probably get rid of this logging now that we are also implementing the same style in my branch too.